website/content/en/preview/testing-guide

```diff
@@ -0,0 +1,41 @@
+# Karpenter Testing User Guide
+
+Currently, users can only test Karpenter by adding to the list of integration tests run in a mock environment or installing and testing Karpenter on a real cluster. Users that want to test more are limited by the lack of automation and configurability, resulting in much less testing.
+
+To increase testing of Karpenter, we will introduce infrastructure and tools to extend the current mechanisms and enable developers to contribute with more confidence.
```

Comment:
Is this referring to the CI infrastructure? If so, let's spell it out.

Reply:
I think it's implied that infrastructure will be CI within the use-cases of the testing detailed below. The design doc will also give more context on this, and what will be CI/CD.